lighthouse-core/audits/content-width.js

```diff
@@ -25,13 +25,21 @@ class ContentWidth extends Audit {
 
   /**
    * @param {LH.Artifacts} artifacts
+   * @param {LH.Audit.Context} context
    * @return {LH.Audit.Product}
    */
-  static audit(artifacts) {
+  static audit(artifacts, context) {
     const viewportWidth = artifacts.ViewportDimensions.innerWidth;
     const windowWidth = artifacts.ViewportDimensions.outerWidth;
     const widthsMatch = viewportWidth === windowWidth;
 
+    if (context.settings.disableDeviceEmulation) {
```

Comment:
wouldn't this be disabled if you're running on a real phone, though?

Reply:
Hm, yes it would. Maybe a `HostUserAgent` plus `disableDeviceEmulation` check? Probably worth rolling up into a generic context `isMobile` boolean

Reply:
fwiw this does also not work for the lr-desktop preset. 

but it looks like we're going to use viewport emulation for lr-desktop now, with viewport of `1366x768` and `mobile:false`, to fix a screenshot bug.

So we also have that situation to contend with. :)